### ISSUE TITLE
About initialization and gan inversion.

### ISSUE THREAD

**Issue by laolongboy** [eladrich/pixel2style2pixel#129]
Hi, I can't find any initialization methods such as orthogonal or kaiming initialization used for psp_encoder. Is this step unnecessary?

And, the script of 'Training the pSp Encoder' describing in README is used for GAN inversion? 

I am trying to use CelebA-HQ images to do inversion, but it seems easy to train for a crash. Empirically, what hyperparameters is this task sensitive to?

Thanks.

**Comment by yuval-alaluf** [comment]
Hi @laolongboy 
For initializing the encoder we use a model pre-trained on the facial recognition task. Notice that we load the parameters from the model irse50. That is why you see no initialization such as kaiming. 
Yes. The section "Training the pSp Encoder" is for inversion. 
Regarding your last question, I am unsure what you mean by easy to crash. We found that the main hyperparameters that affect training are the loss weights. We did play around with them to find a nice balance of the losses, but I am not sure if they are overly sensitive or anything like that.
